baselines/common/vec_env/vec_video_recorder.py

```diff
@@ -0,0 +1,70 @@
+import os
+from baselines import logger
+from baselines.common.vec_env import VecEnvWrapper
+from gym.wrappers.monitoring import video_recorder
+
+
+class VecVideoRecorder(VecEnvWrapper):
+    def __init__(self, venv, directory, video_callable, video_length=200):
+        VecEnvWrapper.__init__(self, venv)
+        self.video_callable = video_callable
```

Comment:
the role of video_callable parameter is not very clear to me... Looks like it is something that defines when to start recording the video, right? Either a more transparent name (maybe start_recording_trigger?)  or a docstring, better yet both :) would be great.

Reply:
Actually, I took this name from code in Gym's wrapper.

https://github.com/openai/gym/blob/520fd3a5e3e1362cf1ab4c03afe90e4e07d47007/gym/wrappers/monitor.py#L13

I added docstring of arguments in [this commit](https://github.com/openai/baselines/pull/666/commits/dd62945fe08acbedebc9c12df589372d1b4cb7a1).
If I should change the name of this argument, please tell me.

Reply:
we are between a rock and a hard place then... I'd say let's change the name to `record_video_trigger` nonetheless. In gym the name is also not particularly illuminating, and also works slightly differently from what I understand (takes number of the episode and returns if it should be recorded or not, whereas here it takes number of step, and returns whether we should start recording or not). Otherwise, LGTM

Reply:
Thank you for suggestion! fixed in https://github.com/openai/baselines/pull/666/commits/99f9b8e14fbcf720596ec7a2c82e84bbd3970791